.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: CLI action
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 16
```

Comment:
use matrix?
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
We currently support Node versions 16, 18, and 20. However, we are considering dropping support for Node version 16 since it has already reached its end of life.